packages/hop-node/src/theGraph/getTransferRootBonded.ts

```diff
@@ -1,17 +1,22 @@
 import makeRequest from './makeRequest'
+import { MaxInt32 } from 'src/constants'
 import { constants } from 'ethers'
 import { normalizeEntity } from './shared'
 
 export default async function getTransferRootBonded (
   chain: string,
   token: string,
+  startDate: number = 0,
+  endDate: number = MaxInt32,
   lastId: string = constants.AddressZero
 ) {
   const query = `
-    query TransferRootBonded($token: String, $lastId: ID) {
+    query TransferRootBonded(${token ? '$token: String, ' : ''}$startDate: Int, $endDate: Int, $lastId: ID) {
```

Comment:
suggestion: since the `token` parameter for the function is required then `$token` here should always be set (no ternary)